### PR TITLE
Don't include "description" in "facts"

### PR DESCRIPTION
**Pull request by trallnag**
### Stop "description" annotation from showing up in card facts 

See #156 for details.

### Changes done to the code base

* Added if statement to exclude "description" annotation to template (both chart and default).
* Incremented chart version.
* Adapted tests by removing "description" fact from expected results and adding if statement
    to the template in test data.

### Checklist
- [x] Code compiles correctly
- [x] Created tests which fail without the change (if possible)
- [x] All tests passing
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

**Comment by Knappek** [comment]
Thanks for the PR @trallnag. 
Please make sure to also adapt the tests such that they succeed and copy the template also over to the [charts default template please](https://github.com/prometheus-msteams/prometheus-msteams/blob/master/chart/prometheus-msteams/card.tmpl).

**Comment by trallnag** [comment]
@Knappek, I think I'm done. I force pushed a rebase, that's why <https://github.com/prometheus-msteams/prometheus-msteams/pull/157#issuecomment-704463395> looks broken

File: pkg/card/testdata/action-message-card.tmpl [hunk 1]
```diff
@@ -25,8 +25,11 @@
       "facts": [
         {{- range $key, $value := $alert.Annotations }}
         {
-          "name": "{{ $key }}",
-          "value": "{{ $value }}"
+          {{- if eq $key "description" -}}
```
Comment (Knappek) [inline pkg/card/testdata/action-message-card.tmpl:28]: see comment above

**Review by Knappek — CHANGES_REQUESTED** [review]

File: default-message-card.tmpl [hunk 2]
```diff
@@ -29,8 +29,11 @@
       "facts": [
         {{- range $key, $value := $alert.Annotations }}
         {
-          "name": "{{ $key }}",
-          "value": "{{ $value }}"
+          {{- if eq $key "description" -}}
+          {{- else -}}
+            "name": "{{ $key }}",
+            "value": "{{ $value }}"
+            {{- end -}}
```
[the remark this answers is not in the document]
Comment (trallnag) [reply default-message-card.tmpl:36]: Thanks for taking your time

File: chart/prometheus-msteams/Chart.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ appVersion: "v1.4.1"
 description: A Helm chart for Kubernetes
 name: prometheus-msteams
 home: https://github.com/prometheus-msteams/prometheus-msteams
-version: 0.6.0
+version: 0.6.1
```
Comment (Knappek) [inline chart/prometheus-msteams/Chart.yaml:6]: you don't need to bump this again. Please use `0.6.0`.

**Review by Knappek — CHANGES_REQUESTED** [review]

File: chart/prometheus-msteams/Chart.yaml (hunk 3 above)
Comment (Knappek) [reply chart/prometheus-msteams/Chart.yaml:6]: Hang on...please don't change this version at all. I will create a new release and bump it, as `appVersion` will also change.

**Review by Knappek — APPROVED** [review]

**Comment by Knappek** [comment]
Thanks again for this PR @trallnag . I will merge it.

**Pull request merged**